manifests/params.pp

```diff
@@ -131,25 +131,28 @@
   $repoforge_exclude = absent
 
   # Drush7 Settings.
-  $drush7_url = "http://pkg.tag1consulting.com/drush/drush-7/el${::operatingsystemmajrelease}/noarch"
+  $drush7_url = "http://packagecloud.io/tag1consulting/drush7/el/${::operatingsystemmajrelease}/${::architecture}"
   $drush7_enabled = '1'
   $drush7_includepkgs = absent
   $drush7_exclude = absent
-  $drush7_gpgcheck = '1'
+  $drush7_gpgcheck = '0'
+  $drush7_repo_gpgcheck = '1'
 
   # Drush8 Settings.
-  $drush8_url = "http://pkg.tag1consulting.com/drush/drush-8/el${::operatingsystemmajrelease}/noarch"
+  $drush8_url = "http://packagecloud.io/tag1consulting/drush8/el/${::operatingsystemmajrelease}/${::architecture}"
```

Comment:
Why are these in an architecture-specific directory if they're all noarch packages?

Reply:
Packagecloud doesn't really have directories, its more of a hash table situation as far as I can see. Their repo URIs always have basearch for RPM distros.